### PR TITLE
(chore): Add my personal script we use in TET and Ingest team to generate transactions [TET-678]

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Usage:
```
bin/send_transactions.py
usage: send_transactions.py [-h] [--dsn http://foo@localhost:3001/11] [--transaction-count TRANSACTION_COUNT] [--transaction-name TRANSACTION_NAME]
                            [--release RELEASE] [--environment ENVIRONMENT]

Send transactions to sentry.

optional arguments:
  -h, --help            show this help message and exit
  --dsn http://foo@localhost:3001/11
                        Sentry project DSN
  --transaction-count TRANSACTION_COUNT
                        Transactions count
  --transaction-name TRANSACTION_NAME
                        Transactions name to send
  --release RELEASE     Release
  --environment ENVIRONMENT
                        Env
```

```
export SENTRY_DSN="http://dummy@localhost:8000/1"
./bin/send_transactions.py --transaction-count=100 --transaction-name="transaction"
INFO:root:Init sentry SDK with {'traces_sample_rate': 1.0, 'release': 'python-demo@4.0.1'}
INFO:root:Sending transaction `/transaction` 1, time now: 2023-03-01 18:08:33.325202
```

### CODE REVIEW

File: bin/send_transactions.py
Comment (jjbayer) [inline bin/send_transactions.py:61]: Does it even make sense to set a default here?

**Review by iambriccardo — APPROVED** [review]

**Comment by iambriccardo** [comment]
Nice!

**Review by jjbayer — APPROVED** [review]
LGTM, but I'd wait for @getsentry/owners-sentry-dev approval to see if even makes sense to place this script in `/bin`.

File: bin/send_transactions.py
Comment (andriisoldatenko) [reply bin/send_transactions.py:61]: good catch, i'll remove it. Agree with you, but not sure who should I ping?

**Review by RaduW — APPROVED** [review]

**Comment by asottile-sentry** [comment]
I'm -1 -- we should not include personal scripts in the `sentry` repository -- they're not up to the quality standards of the rest of the repository, they don't have tests, they're not documented, they're not being linted, they're dead code to most users

**Comment by andriisoldatenko** [comment]
> I'm -1 -- we should not include personal scripts in the `sentry` repository -- they're not up to the quality standards of the rest of the repository, they don't have tests, they're not documented, they're not being linted, they're dead code to most users

@asottile-sentry it's not a personal script, I think at least two teams are using it currently. Other bin scripts don't have tests also. Regarding linting, I think we run all python linters against this file.

**Comment by asottile-sentry** [comment]
> > I'm -1 -- we should not include personal scripts in the `sentry` repository -- they're not up to the quality standards of the rest of the repository, they don't have tests, they're not documented, they're not being linted, they're dead code to most users
> 
> @asottile-sentry it's not a personal script, I think at least two teams are using it currently. Other bin scripts don't have tests also. Regarding linting, I think we run all python linters against this file.

the other scripts in bin are being used by the devtools, build.  my recommendation yesterday to you in slack is to make a repository for your team

**Comment by andriisoldatenko** [comment]
> > > I'm -1 -- we should not include personal scripts in the `sentry` repository -- they're not up to the quality standards of the rest of the repository, they don't have tests, they're not documented, they're not being linted, they're dead code to most users
> > 
> > 
> > @asottile-sentry it's not a personal script, I think at least two teams are using it currently. Other bin scripts don't have tests also. Regarding linting, I think we run all python linters against this file.
> 
> the other scripts in bin are being used by the devtools, build. my recommendation yesterday to you in slack is to make a repository for your team

cool will do that.

**Pull request closed without merging**
